library/common/config_template.cc

```diff
@@ -28,6 +30,46 @@ R"(
             - {{ domain }}
       sni: {{ domain }}
     type: LOGICAL_DNS
+  - name: base_wlan
```

Comment:
It's not ideal that most config changes to these clusters will now need to be made in 3 places (wwan/wlan/base) - is there any way we can share these within the file? Not a blocker, but it'd be a nice improvement

Reply:
Great callout. I can factor out some of the common configuration in a follow up PR, if that sounds alright.